oneflow/core/control/rank_info_bootstrap_server.cpp

```diff
@@ -43,10 +59,59 @@ RankInfoBootstrapServer::RankInfoBootstrapServer(const BootstrapConf& bootstrap_
   server_builder.RegisterService(grpc_service_.get());
   cq_ = server_builder.AddCompletionQueue();
   grpc_server_ = server_builder.BuildAndStart();
-  if (bootstrap_conf.rank() == 0) { CHECK_EQ(p, port()) << "Port " << p << " is unavailable"; }
+  if (rank == 0) { CHECK_EQ(p, port()) << "Port " << p << " is unavailable"; }
   LOG(INFO) << "RankInfoBootstrapServer listening on "
             << "0.0.0.0:" + std::to_string(port());
   loop_thread_ = std::thread(&RankInfoBootstrapServer::HandleRpcs, this);
+  if (rank == 0) {
+    rank2host_ = std::make_shared<std::vector<std::string>>(world_size_);
```

Comment:
```suggestion
    rank2host_ = std::make_shared<std::vector<std::string>>(world_size_, "");
```
因为后面用到了vector中的默认值，所以这里最好显示制定一下构感vector 时的默认值。

Reply:
好的，已修改